src/treemodel/node.js

```diff
@@ -188,6 +188,9 @@ export default class Node {
 		// Due to circular references we need to remove parent reference.
 		json.parent = this.parent ? this.parent.name : null;
 
+		// Serialize attributes as Map object is represented as "{}" when parsing to JSON.
+		json._attrs = [ ...json._attrs ];
```

Comment:
It is not a part of this ticket I think. While I can change this to:

```js
json._attributes = [ ...json._attrs ];
```
I also need to remove:
```js
delete json._attrs;
```

tl;dr - this name mirrors object's properties.